modules/react/action-bar/lib/ActionBar.tsx

```diff
@@ -1,76 +1,55 @@
+import {Globals} from 'csstype';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {commonColors, colors, space} from '@workday/canvas-kit-react/tokens';
+import {HStack, HStackProps} from '@workday/canvas-kit-labs-react';
 
-export interface ActionBarProps extends React.HTMLAttributes<HTMLDivElement> {
+type PropertyPosition =
+  | Globals
+  | '-webkit-sticky'
+  | 'absolute'
+  | 'fixed'
+  | 'relative'
+  | 'static'
+  | 'sticky';
+
+interface ActionBarProps {
   /**
-   * If true, fix the ActionBar to the bottom of the screen.
-   * @default false
+   * Sets the position for the container
+   * @default "fixed"
    */
-  fixed?: boolean;
-}
-
-function getFixedStyles(fixed = false): CSSProperties {
-  return fixed
-    ? {
-        position: 'fixed',
-        left: 0,
-        bottom: 0,
-        right: 0,
-      }
-    : {};
+  position?: PropertyPosition;
 }
 
-const ActionBarContainer = styled('div')<ActionBarProps>(
-  {
-    borderTop: `solid 1px ${colors.soap400}`,
-    background: commonColors.background,
+const ResponsiveHStack = styled(HStack)<HStackProps & StyledType>(({theme}) => ({
+  [theme.canvas.breakpoints.down('s')]: {
     padding: space.s,
-    boxShadow: '0 -2px 4px rgba(0, 0, 0, 0.08)',
-  },
-  ({fixed, theme}) => {
-    return {
-      ...getFixedStyles(fixed),
-      [theme.canvas.breakpoints.down('s')]: {
-        padding: space.xxs,
-      },
-    };
-  }
-);
-
-const ChildrenContainer = styled('div')(
-  {
-    display: 'inline-block',
-    padding: `0 ${space.m}`,
-    '*:not(:first-of-type)': {
-      marginLeft: space.s,
+    '> *:not(div)': {
+      flex: 1,
     },
   },
-  ({theme}) => ({
-    [theme.canvas.breakpoints.down('s')]: {
-      display: 'flex',
-      padding: space.xxs,
-      justifyContent: 'center',
-      flexDirection: 'row-reverse',
-      '> *': {
-        flex: 1,
-        '&:not(:first-of-type)': {
-          marginRight: space.s,
-          marginLeft: 0,
-        },
-      },
-    },
-  })
-);
+}));
 
-export default class ActionBar extends React.Component<ActionBarProps> {
-  public render() {
-    const {fixed, children, ...elemProps} = this.props;
+const containerStyles: Omit<HStackProps, 'spacing'> = {
+  background: commonColors.background,
+  borderTop: `solid 1px ${colors.soap400}`,
+  bottom: 0,
+  depth: 1,
+  left: 0,
+  padding: `${space.s} ${space.xl} `,
+  width: '100%',
+};
 
-    return (
-      <ActionBarContainer {...elemProps} fixed={fixed}>
-        <ChildrenContainer>{children}</ChildrenContainer>
-      </ActionBarContainer>
-    );
-  }
-}
+export const ActionBar = createComponent('div')({
+  displayName: 'ActionBar',
+  Component: ({position = 'fixed', ...elProps}: ActionBarProps, ref, Element) => (
+    <ResponsiveHStack
+      ref={ref}
+      as={Element}
+      position={position}
+      spacing="s"
+      {...containerStyles}
+      {...elProps}
+    />
+  ),
+});
```

Comment:
I'm not sure there's a need for `ActionBarProps`. If so, I think we could more consistent with default props vs overriding props. For example, `spacing="s"` is an override prop while `position={position}` is a default prop. Also, we use `elemProps` everywhere else instead of `elProps`. I don't have a problem with `elProps`, but is there a strong reason to go with `elProps` instead?

```suggestion
export const ActionBar = createComponent('div')({
  displayName: 'ActionBar',
  Component: (elemProps: ActionBarProps, ref, Element) => (
    <ResponsiveHStack
      ref={ref}
      as={Element}
      position="fixed"
      spacing="s"
      {...elemProps}
    />
  ),
});
```